toolkit.nu

```diff
@@ -274,4 +316,91 @@ export def install [
     cargo install --path . --features ($features | str join ",")
 }
 
+def windows? [] {
+    $nu.os-info.name == windows
+}
+
+# filter out files that end in .d
+def keep-plugin-executables [] {
+    if (windows?) { where name ends-with '.exe' } else { where name !~ '\.d' }
+}
+
+# register all installed plugins
+export def "plugin register" [] {
```

Comment:
How about naming it this?
```suggestion
export def "register plugins" [] {
```

Reply:
no strong preference on this :+1: 
that looks more like the original script so let's go with this :wink:

Reply:
done in e00323e56372a81bf494f05eb51b03a005d5451f